lib/fruitstrap.js

```diff
@@ -0,0 +1,52 @@
+import { exec } from 'teen_process';
+import logger from './logger';
+
+const FRUITSTRAP_PATH = 'fruitstrap';
+
+class Fruitstrap {
+ 
+  constructor (udid, fruitstrapPath) {
+    this.udid = udid;
+    this.fruitstrapPath = fruitstrapPath || FRUITSTRAP_PATH;
```

Comment:
I wanted to throw an exception here if path is not valid. Is there trick to call `fs.hasAcces` an async function in constructor?